app/questionnaire/questionnaire_schema.py

```diff
@@ -248,6 +248,15 @@ def get_relationship_collectors_by_list_name(self, list_name: str):
                 if block["for_list"] == list_name
             ]
 
+    def get_list_name(self, current_location):
```

Comment:
I think this might be clearer if this was named something like `get_list_name_for_location`.